task/banking.ts

```diff
@@ -0,0 +1,164 @@
+// defining interfaces.
+interface User {
+    id: number;
+    name: string;
+    email: string;
+    password: string;
+}
+
+interface Transaction {
+    id: number;
+    userId: number;
+    amount: number;
+    type: 'withdrawal' | 'deposit' | 'transfer';
+}
+
+interface Account {
+    id: number;
+    userId: number;
+    balance: number;
+}
+
+// defining class for different operations
+class Bank {
+    private users: User[] = [];
+    private transactions: Transaction[] = [];
+    private accounts: Account[] = [];
+
+    createUser(user: User): void {
+        const newUser: User = {
+            id: user.id,
+            name: user.name,
+            email: user.email,
+            password: user.password,
+        };
+        this.users.push(newUser);
+    }
+
+    createAccount(userId: number): void {
+        const newAccount: Account = {
+            id: this.accounts.length + 1,
+            userId,
+            balance: 0,
+        };
+        this.accounts.push(newAccount);
```

Comment:
In the `createAccount` method, you can directly push the `account` parameter into the accounts array instead of creating a new object newAccount.